src/init.cpp

```diff
@@ -69,6 +70,7 @@ static const bool DEFAULT_REST_ENABLE = false;
 static const bool DEFAULT_DISABLE_SAFEMODE = false;
 static const bool DEFAULT_STOPAFTERBLOCKIMPORT = false;
 
+std::shared_ptr<CConnman> g_connman;
```

Comment:
I don't see g_connman requiring any refcounting, could this be a plain unique_ptr?